docs/develop/go/service-communication.mdx

```diff
@@ -0,0 +1,130 @@
+---
+sidebar_position: 2
+description: "Find out how Restate services can send requests to each other."
+---
+
+import Admonition from '@theme/Admonition';
+
+# Service Communication
+
+A handler can call another handler and wait for the response (request-response), or it can send a message without waiting for the response.
+
+## Request-response calls
+
+Request-response calls are requests where the client waits for the response.
+
+<CH.Scrollycoding className={"single-item"}>
+
+    To a Service:
+
+    ```go
+    CODE_LOAD::go/develop/servicecommunication.go#request_response_service
```

Comment:
All the snippets on this page are too large, and i have an horizontal scroll bar for those. Perhaps can you newline on the .Request/.Send?